components/canvas/canvas_paint_thread.rs

```diff
@@ -127,6 +121,11 @@ impl<'a> CanvasPaintThread<'a> {
                  antialias: bool)
                  -> IpcSender<CanvasMsg> {
         let (sender, receiver) = ipc::channel::<CanvasMsg>().unwrap();
+        let antialias = if antialias {
```

Comment:
I also checked that but `azure::azure_hl::AntialiasMode` isn't available in the crate that uses it `components/constellation/constellation.rs` and I didn't want to change the .toml file for my first PR.